### PR TITLE
Update pytest.ini

### PR DESCRIPTION
**Pull request by Dreamsorcerer**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -74,7 +74,8 @@ jobs:
       env:
         COLOR: 'yes'
       run: |
-        pytest tests --cov-report xml
+        pytest tests
+        python -m coverage xml
```
Comment (Dreamsorcerer) [inline .github/workflows/ci.yml:78]: @webknjaz Not sure if this makes sense to you, but seems that if I use --cov-report, then codecov misses the `__init__.py` files (although they appear in the terminal coverage output).

Comment (webknjaz) [reply .github/workflows/ci.yml:78]: Dunno. It might depend on the args that `pytest.ini` has, or env vars...

Comment (webknjaz) [reply .github/workflows/ci.yml:78]: I don't see any "skip empty" tho

Comment (Dreamsorcerer) [reply .github/workflows/ci.yml:78]: It's not empty, it contains half the project. :P

See codecov summary above, it's 119 lines difference.

Comment (webknjaz) [reply .github/workflows/ci.yml:78]: Suppose I mean "skip covered"

**Pull request merged**